src/EditorFeatures/VisualBasicTest/CodeActions/InlineTemporary/InlineTemporaryTests.vb

```diff
@@ -2916,7 +2916,7 @@ End Module
         End Function
 
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTemporary)>
-        Public Async Function TestEscapeKeywordsIfNeeded5CommentsAfterLineContinuation() As Task
+        Public Async Function TestEscapeKeywordsIfNeeded5CommentsAfterLineContinuation_PC() As Task
```

Comment:
💡 The tests should keep their original names.

```suggestion
        Public Async Function TestEscapeKeywordsIfNeeded5CommentsAfterLineContinuation() As Task
```

Reply:
Yes I put the _PC in to make them easier to find when debugging only unfortunately I forgot to remove them. I will just remove them,